automation_testing/tests/dataview/save_as.spec.ts

```diff
@@ -0,0 +1,105 @@
+import { test, expect } from "@playwright/test";
+import { dataview } from "../../utils/data/dataview_data";
+import { DataviewPage } from "../../pages/DataViewPage";
+import { SaveAsComponent } from "../../pages/SaveAsComponent";
+
+test.describe("Data View", () => {
+
+	let dataviewPage: DataviewPage;
+	let saveAs: SaveAsComponent;
+	test.beforeEach(async ({ page }) => {
+		dataviewPage = new DataviewPage(page);
+		saveAs = dataviewPage.saveAsComponent;
+
+		await dataviewPage.visit();
+	});
+
+	test("Save As", async () => {
+		await dataviewPage.validateSnatshot(saveAs.saveAsBtn, "save_as_btn");
```

Comment:
I believe there's a spelling error on all places this function is being used, which might cause issues in the future (I believe it should be validateSna**p**shot)

Reply:
@mauriciodeleonc  I already fix this comment